roofit/roofitcore/inc/RooAddModel.h

```diff
@@ -94,7 +94,6 @@ class RooAddModel : public RooResolutionModel {
   mutable RooSetProxy _refCoefNorm ;   ///<! Reference observable set for coefficient interpretation
   mutable TNamed* _refCoefRangeName = nullptr;  ///<! Reference range name for coefficient interpretation
 
-  bool _projectCoefs = false;  ///< If true coefficients need to be projected for use in evaluate()
```

Comment:
Will it be fine for the ROOT I/O reading an old file after removing the data member ? Same for RooAddPdf

Reply:
Yes, the removal of data member is no problem. It's handled by the automatic schema evolution